ansible/roles/ocp4-workload-dil-agile-integration/templates/che-devfile.yaml.j2

```diff
@@ -66,7 +67,7 @@ components:
           -Duser.home=/home/user
       - name: MAVEN_OPTS
         value: $(JAVA_OPTS)
-    memoryLimit: 768Mi
+    memoryLimit: 3Gi
```

Comment:
x 3 sounds a big change ?

Reply:
Hi @fridim yes the camel plugin requires 2GB RAM minimum to get started, plus we have camelk, quarkus and java8 plugins. Consistency in startup is only possible with this minimum.

It would not impact on the students/node ratio as the estimate was done with this demand for che.